tests/features/event.feature

```diff
@@ -26,7 +27,7 @@ Feature: Event feature
     And I should see "My description"
     And I should see the link "legal expenses"
     And I should see "fr"
-    And I should see "http://example.com/my-event/more-info"
+    And I should see the link "My related link"
```

Comment:
I would still test that the link actually links to that URL, we could use a step like:
```php
<?php

class MinkContext extends RawMinkContext {

  /**
   * Check that a link pointing to a specific target.
   *
   * @Then I should see the link :link pointing to :url
   *
   * @throws \Exception
   *   If region or link within it cannot be found.
   */
  public function assertLinkRegion($link, $url) {
    $page = $this->getSession()->getPage();

    $result = $page->findLink($link);
    if (empty($result)) {
      throw new ExpectationException("No link '{$link}' on the page", $this->getSession());
    }

    $href = $result->getAttribute('href');
    if ($url != $href) {
      throw new ExpectationException("The link '{$link}' points to '{$href}'", $this->getSession());
    }

  }
```